dev/stats/get_important_pr_candidates.py

```diff
@@ -123,58 +231,84 @@ def __str__(self) -> str:
 
 
 @click.command()
+@option_github_token  # TODO: this should only be required if --load isn't provided
```

Comment:
I attempted to implement a custom click.Option class to accomplish this and burned too much time.